src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -543,28 +543,7 @@ def query_trace_data(
     )
     occurrence_query.groupby = [Column("event_id"), Column("occurrence_id")]
 
-    error_query = QueryBuilder(
-        Dataset.Events,
-        params,
-        query=f"trace:{trace_id}",
-        selected_columns=[
-            "id",
-            "project",
-            "project.id",
-            "timestamp",
-            "trace.span",
-            "transaction",
-            "issue",
-            "title",
-            "tags[level]",
-        ],
-        # Don't add timestamp to this orderby as snuba will have to split the time range up and make multiple queries
-        orderby=["id"],
-        limit=limit,
-        config=QueryBuilderConfig(
-            auto_fields=False,
-        ),
-    )
+    error_query = find_errors_for_trace_id(params, trace_id, limit)
```

Comment:
I'm going to call `find_errors_for_trace_id` from my endpoint, thus, making it its own function.

Reply:
I'd prefer you re-implement this function rather than re-use here so that I can still make changes here without potentially breaking your endpoint. 

Especially because the query pattern here is very dependent on feeding the latter `augment_transactions_with_spans`

Reply:
wfm